docs/reference/garbage-collection.md

```diff
@@ -106,6 +106,18 @@ spark-submit --class io.treeverse.clients.GarbageCollector \
   <APPLICATION-JAR-PATH> \
   example-repo us-east-1
 ```
+
+### End of run reports 
+
+At the end of each GC job, a few files will be added under the repository's storage namespace, containing details about the run.
```

Comment:
We should mention that this report is available from a certain metadata client version (that we haven't released yet). so need to release and mention it here.